Sources/MapboxMaps/Foundation/Attribution/AttributionView.swift

```diff
@@ -0,0 +1,32 @@
+// Consider moving blurring to this view. Unfortunately it doesn't seem
+// possible to use UIVisualEffectView as the base class, as the blurring is not
+// rendered when used with the current Snapshotter implementation.
+//
+// Attribution view has a subview label with attributed text
+internal final class AttributionView: UIView {
+
+    internal static let padding = CGPoint(x: 10, y: 5)
+
+    init(text: NSAttributedString) {
+        // Label
+        let label = UILabel()
```

Comment:
this feels like it should be a property of a class instead of a local variable.

Reply:
At this moment, I don't want to allow the option to modify the label's contents (though this might be something we choose to do in future).

Reply:
fair point!
Nit: but could you remove the comment, seems pointless.